x/axelarnet/types/codec.go

```diff
@@ -54,6 +55,10 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	}
 
 	r.RegisterCustomTypeURL((*sdk.Msg)(nil), "/axelar.axelarnet.v1beta1.ExecuteMessageRequest", &RouteMessageRequest{})
+
+	registry.RegisterImplementations((*govtypes.Content)(nil),
```

Comment:
Amino codec registration is missing above

Reply:
Are you sure it's needed when it's wrapped with proto.Any?

Reply:
It's not needed. The actual content is never serialized/deserialized in the axelarnet module but gov module.